src/components/userinfo.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react'
+import userContext, { UserConsumer } from '../app/usercontext'
+
+const UserInfo = () => {
+  
+  const { users } = useContext(userContext)
+
+  const loggedin = users.user.loggedin
```

Comment:
as a good practices, you can use question forms for booleans in the naming
for example:
`isLoggedin`
`hasLoggedin`
etc...